src/Components/ModelledPropertyBuilder/ModelledPropertyBuilder.types.ts

```diff
@@ -169,6 +169,15 @@ export const numericPropertyValueTypes: PropertyValueType[] = [
     'long'
 ];
 
+export const queryAllowedPropertyValueTypes: PropertyValueType[] = [
```

Comment:
name should be all caps since it's a const.

Reply:
I'm following convention of other consts in that file, ModelledPropertyBuilder.types.ts. I would prefer not to change all of them to all caps.

Reply:
hmmm. I'm biased. I would rather know something is a constant and not a variable. I wouldn't chang ethe rest, but I would question whether this const is really for the ModeledPropertyBuilder since your component is separate from that. Shouldn't this be out in your own file for your component?

Reply:
There are shared consts for propertyValueTypes in this file, wanted to follow that pattern as well.

Reply:
we should not be cross referencing between components. Please pull those shared consts out to a common place and use them there.